tools/scripts-config-jest/jest.config.js

```diff
@@ -36,10 +36,10 @@ const d3Pkgs = [
 ];
 
 // option 1 map module to an bundled version of the package which is es5
-const moduleNameMapper = d3Pkgs.reduce((acc, pkg) => {
-	acc[`^${pkg}$`] = path.join(require.resolve(pkg), `../../dist/${pkg}.min.js`);
-	return acc;
-}, {});
+// const moduleNameMapper = d3Pkgs.reduce((acc, pkg) => {
```

Comment:
We want to keep it as comment ?

Reply:
As you wish, this part was really tricky to support es6 import by jest so do you want documentation on it or not ?

Reply:
I don't use it personally so i cannot say if we need to keep track of it or not , let's say we can keep it there if someone needs some info for its own project